docs/diginstroom/sip/2.1/profiles/film.md

```diff
@@ -72,31 +73,142 @@ root_directory
   - The value of element `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm` MUST be set to `MD5`.
   - The value of attribute `premis:premis/premis:object[@xsi:type="premis:file"]/premis:objectCharacteristics/premis:fixity/premis:messageDigestAlgorithm/@valueURI` MUST be set to `"http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/md5"`.
   - The value of all `//*/@CHECKSUMTYPE` attributes in the `METS.xml` files MUST be set to `MD5`.
-- <mark>There MAY be descriptive metadata at the representation level (e.g. information about the representations, such as a title or a description).</mark>
+- There SHOULD be descriptive metadata about the carrier(s) (i.e. the reel(s)) at the package level in the `preservation/premis.xml` file.
```

Comment:
This is more clear now.

However, it might be also more clear to state that descriptive metadata of the IE MUST be specified on the package level.

Reply:
Added this as an another requirement.